service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/azure/vm/ControlledAzureVmHandler.java

```diff
@@ -31,6 +31,7 @@ public WsmResource makeResourceFromDb(DbResource dbResource) {
             .ipId(attributes.getIpId())
             .networkId(attributes.getNetworkId())
             .diskId(attributes.getDiskId())
+            .customScriptExtension(attributes.getCustomScriptExtension().orElse(null))
```

Comment:
It seems like using `Optional` here does not provide much value, a `null` value is valid.

Reply:
It is inconsistent. The getter from the resource class does not return an optional, but the getter from the attributes does.

Reply:
That was my initial thought. Forgot to remove it.